### ISSUE TITLE
iOS VoiceOver accessibility fails because of `role="textbox"`

### ISSUE THREAD

**Issue by naerymdan** [surveyjs/survey-library#6058]
### Are you requesting a feature, reporting a bug or asking a question?
Reporting a bug

### What is the current behavior?
Active fields inside a question is not made available to VoiceOver in iOS

### What is the expected behavior?
Pure blind swiping on iOS with VoiceOver should allow a user to answer all question types.

### How would you reproduce the current behavior (if this is a bug)?
1. Using an iOS device, Enable VoiceOver
2. Swipe right to advance through the survey (you cannot single-click on anything, close your eyes, pretend to be blind)
3. Double-tap to "activate" stuff.

#### Provide the test code and the tested page URL (if applicable)
Not working:
 - Text fields: https://surveyjs.io/form-library/examples/s/text-entry-question/reactjs
 - NPS: https://surveyjs.io/form-library/examples/s/nps-question/reactjs

Working example: 
 - Radio buttons: https://surveyjs.io/form-library/examples/s/how-to-configure-and-embed-product-market-fit-survey/reactjs
 - Text field (navigate till example field): https://www.w3.org/WAI/tutorials/forms/instructions/

### Specify your

* browser: Safari iOS
* browser version: 16.4.1 (latest)
* surveyjs platform (angular or react or jquery or knockout or vue): react & knockout (possibly others)
* surveyjs version: 1.9.84

## Edit - Additional details

The Chrome "Accessibility Tree View" reveals that the question div has a role of textbox which, once removed, allows iOS VoiceOver to correctly work. Working question types, such as radiogroup do not have the textbox role added to the question div.
```html
<div id="sq_107" class="sd-element--nested sd-element sd-question sd-row__question" role="textbox" aria-required="false" aria-invalid="false" aria-labelledby="sq_107_ariaTitle" data-rendered="r" data-name="nps-score">
```

Adding the following awful workaround make VoiceOver work on most question that previously failed:
```javascript
window.survey.onAfterRenderSurvey.add(function (sender, options) {
        const removeTextboxRoles = function () {
            document.querySelectorAll("[role=textbox]").forEach(function(el) {
                el.removeAttribute("role");
            });
        };
        // Need timeout otherwise the document does not have the question divs yet
        setTimeout(removeTextboxRoles, 1);
    });
```

**Comment by JaneSjs** [comment]
Hello @naerymdan,
We may need additional time to reproduce the issue. I'll update this thread once I have any news or require additional clarification.

Thanks

**Comment by dmitrykurmanov** [comment]
We tried on Mac. We will try to reproduce the problem on Ipad.

**Comment by JaneSjs** [comment]
Hello @naerymdan,
We reproduced this issue on an iPad. We will research this issue and update this thread once we have any news to share.

Please stay tuned.
